packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,8 +170,94 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  //Whether multiple panels can be open simultaneously
+  final bool _allowMultiplePanelsOpen;
+
+  @override
+  State<StatefulWidget> createState() => new _ExpansionPanelListState();
+}
+
+class _ExpansionPanelListState extends State<ExpansionPanelList> {
+  Map<int, bool> _widgetOpenMap;
+
+  @override
+  void initState() {
+    super.initState();
+
+    if (!widget._allowMultiplePanelsOpen) {
+      _widgetOpenMap = <int, bool>{};
```

Comment:
Initialize this where it is declared, and rename it to describe what it is for instead of what it is.

`Map<int, bool> _openPanels = <int, bool>{}`